fileshare/libdrop/libdrop.go

```diff
@@ -167,46 +293,46 @@ func (f *Fileshare) Accept(transferID, dstPath string, fileID string) error {
 	f.mutex.Lock()
 	defer f.mutex.Unlock()
 
-	res := f.norddrop.Download(transferID, fileID, dstPath)
-	if err := toError(res); err != nil {
+	if err := f.norddrop.DownloadFile(transferID, fileID, dstPath); err != nil {
 		return err
 	}
 
 	return nil
 }
 
-// Cancel file transfer.
-func (f *Fileshare) Cancel(transferID string) error {
+// Finalize file transfer.
+func (f *Fileshare) Finalize(transferID string) error {
 	f.mutex.Lock()
 	defer f.mutex.Unlock()
-
-	res := f.norddrop.CancelTransfer(transferID)
-	return toError(res)
+	return f.norddrop.FinalizeTransfer(transferID)
 }
 
 // CancelFile id in a transfer
 func (f *Fileshare) CancelFile(transferID string, fileID string) error {
 	f.mutex.Lock()
 	defer f.mutex.Unlock()
 
-	res := f.norddrop.RejectFile(transferID, fileID)
-	if err := toError(res); err != nil {
+	if err := f.norddrop.RejectFile(transferID, fileID); err != nil {
```

Comment:
can just `return f.norddrop.RejectFile(transferID, fileID)`?

Reply:
Yup. Fixed (I'll push the fixes for it and the rest of the comments tomorrow)